elections/ArviZ_2020.md

```diff
@@ -0,0 +1,44 @@
+# Election 2020
+
+## Summary of Process
+This is the first election cycle for ArviZ. During this cycle we are aiming
+for the following outcomes
+
+* Ratification of Governance.MD
+* Identification and listing of Core Contributors
+* Nominations for Council Candidates 
+* Council Elections. Particularly
+  * Size of Council
+  * Inaugral Arviz Council
+
+
+## Process Details
+### Identification of Core Contributors (1 week)
+1. Google Form will be created to get core contributor names and emails
+    * Emails are needed for Elections steps so NumFOCUS can mail and track votes
+    * Google Form is moderated by Ravin
+2. At conclusion of week Ravin will update PR with all names and make forms responses public,
+with exception of email
+
+
+### Nominations for Council (1 week)
+1. A github issue labeled *Arviz 2020 Council Elections* will be opened for nominations
+2. Anyone can make a nominate a core contributor (including general public)
+3. Core Contributor must accept acknowledgement with thumbs up on nomination
```

Comment:
If he doesn't, this means his name isn't considered for Council Election, right? Maybe add this precision?

Reply:
Their name isn't considered because we don't know if they willingness or time to exercise the role

Reply:
I added a note for that